docs/sql-error-conditions-invalid-format-error-class.md

```diff
@@ -49,6 +49,10 @@ The escape character is not allowed to end with.
 
 The escape character is not allowed to precede `<char>`.
 
+## MISMATCH_INPUT
+
+The input <inputType> '<input>' does not match the format.
```

Comment:
I think we should follow our convention and don't perform quoting in error formats.
```suggestion
The input <inputType> <input> does not match the format.
```

Reply:
Okay.